### PR TITLE
apply single ulimit as soft+hard limit

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Applied "single" ulimit as both hard and soft

**Related issue**
close https://github.com/docker/compose-cli/issues/1638
https://docker.atlassian.net/browse/IL-135

### CODE REVIEW

**Comment by ndeloof** [comment]
This is a typical case where I miss ternary operator in Go

File: local/compose/create.go [hunk 1]
```diff
@@ -438,10 +438,18 @@ func getDeployResources(s types.ServiceConfig) container.Resources {
 	}
 
 	for name, u := range s.Ulimits {
+		soft := u.Single
+		if u.Soft != 0 {
+			soft = u.Soft
+		}
+		hard := u.Single
+		if u.Hard != 0 {
+			hard = u.Soft
```
Comment (gtardif) [inline local/compose/create.go:447]:
```suggestion
			hard = u.Hard
```

Comment (ndeloof) [reply local/compose/create.go:447]: Stupid me :'(

**Review by lorenrh — APPROVED** [review]
🎉

**Pull request merged**